src/electron-main.ts

```diff
@@ -119,8 +119,9 @@ async function tryPaths(name: string, root: string, rawPaths: string[]): Promise
 
 const homeserverProps = ["default_is_url", "default_hs_url", "default_server_name", "default_server_config"] as const;
 
-// Find the webapp resources and set up things that require them
-async function setupGlobals(): Promise<void> {
+async function loadConfig(): Promise<void> {
```

Comment:
doc comment, please. what does this do? Also worth calling out that it is a no-op if the config has already been read.